care/utils/serializer/tests/test_phonenumber_ispossible_field.py

```diff
@@ -0,0 +1,27 @@
+from unittest import TestCase
+
+from rest_framework.exceptions import ValidationError
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.exceptions'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5343021)